sdk/core/Azure.Core/tests/NextLinkOperationImplementationTests.cs

```diff
@@ -12,23 +12,23 @@ namespace Azure.Core.Tests
 {
     public class NextLinkOperationImplementationTests
     {
-        [Test]
-        public void ConstructRehydrationToken()
-        {
-            var requetMethod = RequestMethod.Get;
-            var startRequestUri = new Uri("https://test");
-            var nextRequestUri = "nextRequestUri";
-            var headerSource = "None";
-            string lastKnownLocation = null;
-            var finalStateVia = OperationFinalStateVia.OperationLocation.ToString();
-            var token = NextLinkOperationImplementation.GetRehydrationToken(requetMethod, startRequestUri, nextRequestUri, headerSource, lastKnownLocation, finalStateVia);
-            Assert.AreEqual(requetMethod, token.RequestMethod);
-            Assert.AreEqual(startRequestUri, token.InitialUri);
-            Assert.AreEqual(nextRequestUri, token.NextRequestUri);
-            Assert.AreEqual(headerSource, token.HeaderSource);
-            Assert.AreEqual(lastKnownLocation, token.LastKnownLocation);
-            Assert.AreEqual(finalStateVia, token.FinalStateVia);
-        }
+        //[Test]
```

Comment:
We can add this back for test coverage, but we should have it use the method that takes the full list of parameters if that makes sense.  It doesn't make sense to me that we would add new methods just to support tests, instead of testing the code paths used by Azure.Core and clients.